### PR TITLE
fix the path issues

### PR DESCRIPTION
**Pull request by Elli610**
related to #207

### CODE REVIEW

File: packages/massa-proto-cli/src/protobuf.ts
Comment (0xSwapFeeder) [inline packages/massa-proto-cli/src/protobuf.ts:91]: Wait for my PR to be merged, #222, already added tihs field with another name, once my branch is merge rebase on main (also true for other PR on massa-proto-cli)

**Review by 0xSwapFeeder — CHANGES_REQUESTED** [review]

**Review by 0xSwapFeeder — APPROVED** [review]

**Review by Ben-Rey — APPROVED** [review]

File: packages/massa-proto-cli/src/protobuf.ts
Comment (gregLibert) [inline packages/massa-proto-cli/src/protobuf.ts:62]: Don't you think we should stop the execution here by throwing an error ?

File: packages/massa-proto-cli/src/ts-gen.ts
Comment (gregLibert) [inline packages/massa-proto-cli/src/ts-gen.ts:27]: Why not throwing an exception instead? Why should we silently rename the file? Is the .proto extension missing an expected behavior?

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: packages/massa-proto-cli/src/ts-gen.ts
Comment (Elli610) [reply packages/massa-proto-cli/src/ts-gen.ts:27]: The goal here is to detect if the name of the file. proto has been modified and no longer corresponds to the name of the function that its content represents in order to generate a helper corresponding to the correct name of the SC function.

Afterwards, renaming the proto file is not a good idea. It would better to rename the helper that is generated rather than the proto file itself

**Review by gregLibert — APPROVED** [review]

**Pull request merged**
